changedetectionio/model/Watch.py

```diff
@@ -210,7 +210,7 @@ def history(self):
         # Read the history file as a dict
         fname = os.path.join(self.watch_data_dir, "history.txt")
         if os.path.isfile(fname):
-            logging.debug("Reading history index " + str(time.time()))
+            logger.debug(f"Reading history index {str(time.time())}")
```

Comment:
```suggestion
            logger.debug(f"Reading watch history index")
```

Loguru includes a correct timestamp